migrate-to-chimera

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+arg=$1
+if [ $# -gt 1 ] ; then
+    echo "This script get at most one argument"
+    echo "Usage:"
+    echo "$0 [-l|--logout]"
+    echo ""
+    echo "  -l | --logout   Logout after migration"
+fi
+
+if ! [ -d $HOME/.config/steam-buddy ] && \
+    ! [ -d $HOME/.local/share/steam-buddy ] && \
+    ! [ -d $HOME/.cache/steam-buddy ] ;
+then
+    # Nothing to migrate, we can exit
+    echo "Found nothing to migrate"
+    exit 0
+fi
+
+## MIGRATION MAGIC ##
+
+# Check if there are config files from steam-buddy
+if [ -d $HOME/.config/steam-buddy ] ; then
+    if [ -d $HOME/.config/chimera ] ; then
+        cp -Rbv $HOME/.config/steam-buddy/* $HOME/.config/chimera/.
+        rm -rfv $HOME/.config/steam-buddy
+        echo "Configuration directories have been merged and existing chimera files been backed up"
+    else
+        mv $HOME/.config/steam-buddy $HOME/.config/chimera
+        echo "Configuration directory has been moved"
+    fi
+fi
+
+# Migrate user data directories and files from steam-buddy
+if [ -d $HOME/.local/share/steam-buddy ] ; then
+    if [ -d $HOME/.local/share/chimera ] ; then
+        cp -Rbv $HOME/.local/share/steam-buddy/* $HOME/.local/share/chimera/.
```

Comment:
I can see this easily failing for me. I have A LOT of large ROMs here and not much disk space left. I think we need to use `mv`.

Reply:
Mhhh a sensible option would be to mv the content parts (that chimera wont have even if it run once) and copy the leftovers with backup option.
But that is really a problem with settings, since we can migrate settings to anothe special location, we can just move and don't backup anything that is already there.

Reply:
I am not sure I follow. I think we should keep it simple. Check if the folder is there, if it isn't move it. Otherwise do nothing.

Reply:
I meant:
- Migrate `settings.json` with `cp -b` thus using steam-buddy configuration currently used by the user (backing up if the file already exist).
- Banners, content, recordings and all other files, just move them over overwiting if they exist. We are supporting the state where the user is migrating from steam-buddy to chimera and not the other way around.